protocol/0056-REWA-rewards_overview.md

```diff
@@ -259,12 +259,34 @@ Recurring transfers can target groups of markets, or all markets for a settlemen
 
 All rewards are distributed to [vesting accounts](./0085-RVST-rewards_vesting.md) at the end of each epoch *after* [recurring transfers](0057-TRAN-transfers.md) have been executed. Funds distributed to the vesting account will not start vesting until the [`lock period`](./0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts) defined in the recurring transfer has expired.
 
-The entire reward account balance is paid out every epoch unless the total value of the metric over all entities is zero, in which case the balance will also be zero anyway (there are no fractional payouts).
+If a `target_notional_volume` is specified in the [recurring transfer](./0057-TRAN-transfers.md#recurring-transfers-to-reward-accounts), the amount to distribute as rewards is first scaled as per the mechanics defined in section [reward scaling](#reward-scaling). The scaled amount is then paid out unless the total value of the metric over all entities is zero.
```

Comment:
is this correct? i.e. do we transfer the full amount into the reward account and then only distribute the scaled amount? I would have thought we only transfer the scaled amount from the source account into the reward account and still distributing the entire reward account balance.

Reply:
> I would have thought we only transfer the scaled amount from the source account into the reward account and still distributing the entire reward account balance.

This is correct, spec poorly phrased on my part, will update.